crates/ruma-common/src/events/room/power_levels.rs

```diff
@@ -265,6 +238,18 @@ pub struct RoomPowerLevels {
     pub notifications: NotificationPowerLevels,
 }
 
+impl RoomPowerLevels {
+    /// Get the power level of a specific user.
+    pub fn for_user(&self, user_id: &UserId) -> Int {
+        self.users.get(user_id).map_or(self.users_default, |pl| *pl)
+    }
+
+    /// Get the maximum power level of any user.
+    pub fn max(&self) -> Int {
```

Comment:
Wouldn't this the maximum PL for *any* action be? This could be a footgun if, say, upgrading a room is 100, but the admin has removed themselves from level 100 (to level 50 or something), and that'd be the "max" then.

Reply:
This is the maximum power level anybody in the room has, not the maximum power level required to do an arbitrary action.